gotchas/styled-jsx-and-jest.md

```diff
@@ -0,0 +1,63 @@
+# TL;DR (recommended setup)
+
+### Usage of `styled-jsx` `css` and `css.resolve` function
+
+In your code-base, make sure to call the `resolve` function directly, like so:
+
+```javascript
+import { resolve } from 'styled-jsx/css';
+const buttonStyle = resolve`button { color: green; }`;
+```
+
+and not as property of the `css` function, i.e. **do not** do this:
+
+```javascript
+import css from 'styled-jsx/css';
+const buttonStyle = css.resolve`button { color: green; }`;
```

Comment:
Why doesn't this work as a mock for `css.resolve`?

```
function css() { return 'default ' }

css.resolve = function () { return 'resolve' }
css.global = function () { return 'global' }

module.exports = css;
```

Reply:
I wish I knew the reason behind it. The above would have made most sense to me too. All I know is that when implementing it like that, jest will think `css` is undefined.